lisa/tools/nvmecli.py

```diff
@@ -87,3 +95,39 @@ def support_device_format(self, device_name: str) -> bool:
         cmd_result = self.run(f"id-ctrl -H {device_name}", shell=True, sudo=True)
         cmd_result.assert_exit_code()
         return self.__format_device_support in cmd_result.stdout
+
+    def get_namespaces(self) -> List[str]:
+        namespaces_cli = []
+        nvme_list = self.run("list", shell=True, sudo=True)
+        for row in nvme_list.stdout.splitlines():
+            matched_result = self._namespace_cli_pattern.match(row)
+            if matched_result:
+                namespaces_cli.append(matched_result.group("namespace"))
+        return namespaces_cli
+
+
+class BSDNvmecli(Nvmecli):
```

Comment:
Does it use the same installation as Linux?

Reply:
It's preinstalled in BSD

Reply:
You can set can_install to false to prevent installation logic reused.